apps/docs/src/guide/errors/error-codes.md

```diff
@@ -0,0 +1,71 @@
+# Error Codes
+
+Here is a list of the expected error codes the SDK can throw. These error codes are used to help understand the error that has been thrown with potential remedies.
+
+Error Codes | Description
+--- | ---
+ABI_MAIN_METHOD_MISSING | When your ABI does not have a main method. This can be resolved by adding a main method to your ABI.
+ABI_TYPES_AND_VALUES_MISMATCH | When the arguments supplied to the function do not match the minimum required input length. Check that the arguments supplied to the function, match the required type.
+ACCOUNT_REQUIRED | When the account has not been set for the contract.
```

Comment:
```suggestion
ACCOUNT_REQUIRED | When the account has not been set for the contract. This is required for contract deployment.
```

Reply:
```suggestion
ACCOUNT_REQUIRED | When a provider has been supplied to the contract that requires an `Account`. Pass though a `Wallet` or `Account` to the ContractFactory constructor to resolve this issue.
```

Reply:
Will leave to you to commit the suggestion 👍🏻

Reply:
Changed to

### Description
When an [`Account`](../../api/Account/Account.md) is required for an operation. This will usually be in the form of a [`Wallet`](../wallets/index.md)
### Resolution
It could be caused during the deployments of contracts, when an account is required to sign the transaction. This can be resolved by the following the deployment guide [here](../contracts/deploying-contracts.md).

Reply:
Nice 👍🏻